.github/workflows/lint.yml

```
@@ -38,12 +38,11 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
```

actionlint (1.7.8)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)